prow/config/istio-private_jobs/00-clean.yaml

```diff
@@ -0,0 +1,13 @@
+common_opts: &common_opts
```

Comment:
why do we need this if its only used one time?

Reply:
You are right. I originally had all the transforms in a single file. I missed that this particular occurence was used only once.